enterprise_gateway/services/kernels/handlers.py

```diff
@@ -54,13 +54,15 @@ async def post(self):
         if model is not None and "env" in model:
             if not isinstance(model["env"], dict):
                 raise tornado.web.HTTPError(400)
-            # Start with the PATH from the current env. Do not provide the entire environment
-            # which might contain server secrets that should not be passed to kernels.
-            env = {"PATH": os.getenv("PATH", "")}
+
+            # Delete by default PATH from client envs
+            # Keeping this variable may break the remote kernel with a different system than the client.
+            # But if user want to keep this var, it's can be set in inherited_envs
+            model['env'].pop("PATH")
```

Comment:
I think this needs to be removed because the user may have specified `"PATH"` in the set of `client_envs`.  In such cases, that configuration needs to be honored.  If that happens to cause issues, that's the user's issue to resolve.

Reply:
Yes you right,
I delete this too quickly 
Thanks